packages/node/src/integrations/tracing/redis.ts

```diff
@@ -17,6 +20,7 @@ import {
   getCacheOperation,
   shouldConsiderForCache,
 } from '../../utils/redisCache';
+import { truncate } from '@sentry/utils/src';
```

Comment:
this import is not correct, should not have the `/src` part (this probably comes from the editor...)